### ISSUE TITLE
j.minuit_device: qlim makes monitoring useless

### ISSUE THREAD

**Issue by theod** [jamoma/JamomaMax#966]
the 2 qlim objects under [j.receive :/monitor/in] and [j.receive :/monitor/out] ignore a lot of message so we can't trust the monitoring when exploring namespace. 

A comment in the patch explains that they are here to "to avoid print jam and spinning beach ball" but I would propose to remove those qlim objects as if we don't want print jam it would be easiest to set @print to 0.

does it makes sens to you @bltzr, @reno- and @avilleret ?

**Comment by theod** [comment]
or a more generic solution would be to create 2 outlet for in and out monitoring and so users can do what they want with those messages... but this implies extra patching around j.minuit_device...

**Comment by reno-** [comment]
I think @lossius add theses qlim objects, so maybe he has on opinion on that.
But i agree that this is weird to have an incomplete monitoring. Both solutions you suggests are ok for me.

**Comment by bltzr** [comment]
Same here!

Pascal Baltazar

**Comment by lossius** [comment]
I believe I added the qlims because there were so much being posted to the Max window that the patch got unresponsive. And this was with a very simple i-score 0.3 setup controlling only one or two parameters.

**Comment by theod** [comment]
@lossius : yes I also encounter the same trouble without qlim objects but I'm wondering if this is a good solution as it breaks the monitoring feature by posting incomplete informations (I spent some times to understand a bug for example).

**Comment by theod** [comment]
"And this was with a very simple i-score 0.3 setup controlling only one or two parameters."
that is strange because with the j.minuit_device.maxhelp (where there are input, degrade and output models) I don't experience such troubles.

is adding deferlow can solve this ?
